torch/csrc/jit_decomp_interface.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <ATen/core/Tensor.h>
+#include <ATen/core/function_schema.h>
+#include <c10/macros/Export.h>
+
+// NOTE: [Jit Decomposition Interface]
+//
+// For some context of why we need this at all, see NOTE: [forward-mode AD
+// decompositions hack]
```

Comment:
Why are we calling this a hack? Since we're upstreaming the mechanism to PyTorch, I would consider it now a supported alternate method to get forward-mode AD coverage. If we want to keep calling it a hack, then we should align on what needs to happen for it to not be a hack

Reply:
This is a very good point, I don't think we have plans to replace it anytime soon.